src/include/units/quantity_point.h

```diff
@@ -182,6 +191,9 @@ template<QuantityLike Q>
 quantity_point(Q) -> quantity_point<typename quantity_like_traits<Q>::dimension,
   typename quantity_like_traits<Q>::unit, typename quantity_like_traits<Q>::rep>;
 
+template<QuantityPointLike QP>
+explicit quantity_point(QP) -> quantity_point_like_type<QP>;
```

Comment:
Do you know if this is valid or the type should be spelled explicitly here? I am asking because even though gcc and MSVC are fine with it, clang complained about it the last time I checked.

Reply:
To my knowledge, yes. I did find it weird that it worked. And also how trying something like this bit me in my own code. I thought that it perhaps had to do with some DR that clarified how template aliases specializations were replaced with the aliased specialization. However, on the standard it seems a requirement at the grammar level, so it should never have worked.

Reply:
OK, I will open an issue on this one.